Enhance certificate handling instructions (#128270)
Azure/azure-content: articles/connectors/connectors-native-http.md

Reviewer: ```suggestion
ms.date: 03/03/2026
```

<a id="tlsssl-certificate-authentication"></a>

### TLS certificate authentication

> 1. Upload your root certificate (and intermediate certificate) to your logic app. See [Add and manage TLS/SSL certificates in Azure App Service](../app-service/configure-ssl-certificate.md)

Reviewer: - Fixes for formatting and writing convention.
- Can you specify the tab where you should upload your root and intermediate certificates?

<img width="633" height="39" alt="Image" src="https://github.com/user-attachments/assets/2c850600-c7b1-4b5f-9018-79b111c36e04" />

```suggestion

1. In the Azure portal, on your Standard logic app sidebar, under **Settings**, select **Certificates**. Upload your root certificate and intermediate certificate.

   For more information, see [Add and manage TLS/SSL certificates in Azure App Service](../app-service/configure-ssl-certificate.md).
```

   }
   ```

> > [!NOTE]

Reviewer: Make plain text.

```suggestion
```

> >

Reviewer: ```suggestion
```

> > If you have a root certificate and intermediate certificate(s), you need to put them in the same setting separated by commas and no whitespace, like:

Reviewer: ```suggestion
   If you have both the root certificate and intermediate certificate, you must add them to the same setting. Make sure to separate them by using a comma without whitespace, for example:

```

> > 84EC242A4EC7957817B8E48913E50953552DAFA6,6A5C65DC9247F762FE17BF8D4906E04FE6B31819.

Reviewer: Replacing with fictitious approved values:
```suggestion

   AA11BB22CC33DD44EE55FF66AA77BB88CC99DD00,BB22CC33DD44EE55FF66AA77BB88CC99DD00EE11
```
